ts/demo-functions/tsconfig.json

```diff
@@ -1,20 +1,27 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
-    "target": "es6",
-    "resolveJsonModule": true,
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "baseUrl": ".",
```

Comment:
Why do you need this?

Reply:
Set baseUrl as node_modules in all our packages so imports are linked after local install.

Reply:
Let me rephrase. Why not leave baseUrl unspecified? We should always use relative paths for importing modules.